meta/categories.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "name": "Developer Tools",
-    "count": 194,
+    "count": 196,
```

Comment:
You should probably increase by one, not by two (don't assume other PRs will be merged before this one - this will create a conflict anyway).